src/cluster/setup.rst

```diff
@@ -201,7 +203,14 @@ setup the node:
 
 .. code-block:: bash
 
-     curl -X POST -H "Content-Type: application/json" http://admin:password@127.0.0.1:5984/_cluster_setup -d '{"action": "enable_cluster", "bind_address":"0.0.0.0", "username": "admin", "password":"password"}'
+     curl -X POST -H "Content-Type: application/json" http://admin:password@127.0.0.1:5984/_cluster_setup -d '{"action": "enable_cluster", "bind_address":"0.0.0.0", "username": "admin", "password":"password", "node_count":"3"}'
+
+.. warning::
+    The ``node_count`` parameter is used to set the ``[cluster]`` ``n`` value.
+    Even when joining more than 3 nodes into a cluster using this endpoint,
+    it is **strongly** recommended not to set this value higher than 3, as
+    it will directly impact the number of replicas of each document in the
+    cluster.
```

Comment:
Thanks for raising this issue, but I think that behaviour is weird. I approved the original change, and now understand that we were missing something. I’ve addressed the issue in https://github.com/apache/couchdb-setup/pull/14

If that PR is merged, we can leave out this warning. This would be a nice-to-have for 2.1, but not blocking.